build-monitor-acceptance/src/main/java/net/serenitybdd/integration/jenkins/environment/PluginDescription.java

```
@@ -21,6 +23,16 @@ public static PluginDescription of(@NotNull Path pluginAtPath) {
         catch(IOException e) {
             throw new RuntimeException(String.format("Couldn't read the manifest file of '%s'.", pluginAtPath.toAbsolutePath()), e);
         }
```

I think that this line should be removed (e.printStackTrace();)
Is it bad practice if I remove it and just swallow exception?